### PR TITLE
SpawnActorPower no longer forces removal of the spawned actor.

### PR DESCRIPTION
**Pull request by phrohdoh**

### CODE REVIEW

**Comment by penev92** [comment]
Seems reasonable :+1:

File: OpenRA.Mods.Common/Traits/SupportPowers/SpawnActorPower.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class SpawnActorPowerInfo : SupportPowerInfo
 		[Desc("Actor to spawn.")]
 		public readonly string Actor = null;
 
-		[Desc("Amount of time to keep the actor alive in ticks.")]
+		[Desc("Amount of time to keep the actor alive in ticks. Value < 1 means this actor will not remove itself.")]
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/SupportPowers/SpawnActorPower.cs]: Value == 0 (remove immediately) may have valid use cases.

Comment (Phrohdoh) [inline OpenRA.Mods.Common/Traits/SupportPowers/SpawnActorPower.cs]: Valid point. `< 0` it is then.

**Comment by Phrohdoh** [comment]
Updated.

**Comment by Mailaender** [comment]
Solar pulse still works. :white_check_mark:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/3423d2132d8dd48e74b78e4fd274230bde38b497...3356f846e6e6824124782af4e21dc68336d47f95)

**Pull request merged**